.changeset/eighty-jars-explain.md

```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/core": minor
+"@salt-ds/lab": minor
+---
+
+Moved `MultilineInput` from lab to core
```

Comment:
Can you align this to how the other changeset have been done please e.g. Accordion

Reply:
done